resources/texts/change_log.txt

```diff
@@ -1,3 +1,70 @@
+[5.7]
+
+* New features and improvements:
+- Introduced a way that you can open models from Thingiverse from your browser
+- Introduced Universal Cura Project files. They contain models and settings and can be shared with people who have different printers.
+- Introduced settings that define the raft offset for top, middle and bottom raft layers
+- Introduced an analyzing tool in the cura engine to better track slicing crashes, and started fixing the biggest reports coming in.
+- New Primetower option; Sparse, next to the current Bucket, contributed by @wawanbreton
+- Brim only on the inside setting
+- Allowing to define print sequence manually, contributed by @alexandr-vladimirov (Github)
+- Introduced a setting to apply Monotonic ordering to the Raft Top Surface
+- Introduced a Raft Wall Line Count for the Bottom, Middle and Top of the raft
+- The play speed of the nozzle in the preview is now closer to the movement of the printhead
+- Introduced an option to choose Drop Model to Buildplate every time a project file is loaded
+- Introduced DisplayInfoOnLCD Post Processing Script which combines DisplayFilenameAndLayerOnLCD with DisplayProgress, contributed by @GregValiant
+- Introduced alternative camera navigation similar to FreeCAD Trackpad, contributed by @quarnster
+- Introduced a way to set print sequence manually when printing One at a time, contributed by @alexandr-vladimirov
```

Comment:
Duplicate of 'Allowing to define print sequence manually' in line 10

Reply:
[b8e4251](https://github.com/Ultimaker/Cura/pull/18615/commits/b8e4251e6a2593aaeeb40f7ac536b17d36e86ccf)

Reply:
Thanks @fieldOfView for pointing this !